urllib3/util/connection.py

```diff
@@ -77,7 +86,6 @@ def create_connection(address, timeout=socket._GLOBAL_DEFAULT_TIMEOUT,
             err = e
             if sock is not None:
                 sock.close()
-                sock = None
 
     if err is not None:
```

Comment:
So this PR seems to hurt our coverage here: we'll need to fix that at some point.